### PR TITLE
Fixed issues on voters and votespage

### PR DESCRIPTION
**Pull request by Lionqueen94**
## Fixed:

- [x] up and down navigation of the fields on the Voters and Votespage
- [x] Marking the W.201 and W.202 warnings only on the field they are about and not also on the total field

### CODE REVIEW

**Review by praseodym — DISMISSED** [review]

**Review by jorisleker — APPROVED** [review]
cycling through the inputs with up and down arrows and enter now works as expected

**Comment by jorisleker** [comment]
Something did break in the styling I just noticed; all cells now seem to have a 1px padding. See screenshots (zoomed in)

Your PR:
Scherm­afbeelding 2024-08-13 om 07 45 06

Before (main):
Scherm­afbeelding 2024-08-13 om 07 46 38

**Comment by jschuurk-kr** [comment]
@Lionqueen94 Can you please mention the two issues you fixed in the description of the PR?

File: frontend/lib/ui/InputGrid/InputGrid.module.css [hunk 1]
```diff
@@ -1,17 +1,21 @@
 .inputgrid {
   margin-top: 1rem;
   width: 100%;
-  border-spacing: 0;
```
Comment (praseodym) [inline frontend/lib/ui/InputGrid/InputGrid.module.css:4]: Looks like removing this caused the cell padding as mentioned in https://github.com/kiesraad/abacus/pull/255#issuecomment-2285385684.

Comment (Lionqueen94) [reply frontend/lib/ui/InputGrid/InputGrid.module.css:4]: Yes I re-added it, so weird because the browser said this had no effect XD

**Review by praseodym — APPROVED** [review]

**Review by jorisleker — APPROVED** [review]

**Pull request merged**